### DISCUSSION TITLE
Color in white terminal are not readable

### DISCUSSION THREAD

**Discussion by antonmedv** [dundee/gdu#386] [General]
Hi! Nice project!

But for me default colors are not readable =( Grey is way too close to white bg.

<img width="732" alt="image" src="https://github.com/user-attachments/assets/a12957e0-837e-4a35-bc8f-9f85441dff0a">

**Comment by dundee** [comment]
Hi, thanks!

Could you share more details? What OS and terminal are you using? What color pallete?

**Reply by antonmedv** [reply]
Macos. Terminal app (default app) with default theme.

**Reply by dundee** [reply]
Are you sure you have the default color pallete? Mine looks quite different:

<img width="682" alt="Screenshot 2024-10-29 at 21 33 27" src="https://github.com/user-attachments/assets/df28268c-5317-4b80-9dea-13b32f9fb24f">

**Reply by antonmedv** [reply]
Looks like it:
<img width="779" alt="image" src="https://github.com/user-attachments/assets/9035de4b-9334-4a38-be71-6842ace7ebeb">

**Reply by antonmedv** [reply]
I also run it via ssh.

**Reply by dundee** [reply]
I am still not able to replicate your colors.

Do you have some configuration set for gdu? E.g. in `~/.gdu.yaml`? 
What version of Gdu are you using?
